pkgs/tools/misc/osm2pgsql/default.nix

```diff
@@ -45,11 +46,15 @@ stdenv.mkDerivation rec {
     "-DEXTERNAL_FMT=ON"
   ] ++ lib.optional withLuaJIT "-DWITH_LUAJIT:BOOL=ON";
 
+  passthru.tests.version = testers.testVersion {
+    package = finalAttrs.finalPackage;
+  };
```

Comment:
:+1: for adding this [sanity test](https://nixos.org/manual/nixpkgs/stable/#tester-testVersion).